lib/src/view/watch/watch_tab_screen.dart

```diff
@@ -176,11 +187,66 @@ class _WatchScreenState extends ConsumerState<WatchTabScreen> {
 
 Future<void> _refreshData(WidgetRef ref) {
   return Future.wait([
+    ref.refresh(broadcastsProvider.future),
     ref.refresh(featuredChannelsProvider.future),
     ref.refresh(liveStreamersProvider.future),
   ]);
 }
 
+class _BroadcastWidget extends ConsumerWidget {
+  const _BroadcastWidget();
+
+  static const int numberOfItems = 5;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final broadcastList = ref.watch(broadcastsProvider);
+
+    return broadcastList.when(
+      data: (data) {
+        return ListSection(
+          header: const Text('Tournament broadcasts'),
```

Comment:
Missing translation.

Reply:
There is no translation for 'Tournament broadcasts'. Should I just use 'Broadcasts' instead ?